src/protocolsupport/protocol/packet/handler/PSInitialHandler.java

```diff
@@ -218,7 +218,7 @@ protected void processLoginStart() {
 				return;
 			}
 			case PE: {
-				if (isOnlineMode()) {
+				if (BungeeCord.getInstance().config.isOnlineMode() || isOnlineMode()) {
```

Comment:
Not again

Reply:
hmm dont we need this if bungee is in online mode though? isOnlineMode ends up getting set to false because the server connection isnt in offline mode =\

Reply:
It was getting false because of a bug.

Reply:
Shev fixed this now

Reply:
awesome, but it back to just isOnlineMode, works great now!